### PR TITLE
bpo-15749: Fix text formatting in cgitb when display is disabled

### PR DESCRIPTION
**Pull request by remilapeyre**
Co-Authored-By: Aaron Iles <aaron.iles+github@gmail.com>

https://bugs.python.org/issue15749

### CODE REVIEW

File: Lib/test/test_sundry.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def test_untested_modules_can_be_imported(self):
                 else:
                     self.fail('{} has tests even though test_sundry claims '
                               'otherwise'.format(name))
+            import bdb
```
Comment (ZackerySpytz) [inline Lib/test/test_sundry.py]: Why was this import added?
Edit: It seems like it originates from a sloppy application of the original patch.

File: Doc/library/cgitb.rst [hunk 2]
```diff
@@ -42,16 +42,25 @@ analysis.
...
+   If the argument *logdir* is present, the traceback reports are written to
+   files.  The value of *logdir* should be a directory where these files will
+   be placed.
+
+   The optional argument *context* is the number of lines of context to
+   display around the current line of source code in the traceback; this
+   defaults to ``5``.
+
+   If the optional argument *format* is ``"html"``, the output is formatted as
+   HTML.  Any other value forces plain text output. The default value is
```
Comment (mangrisano) [inline Doc/library/cgitb.rst]: Please, add another blank space after the last full stop.

File: Doc/library/cgitb.rst [hunk 3]
```diff
@@ -42,16 +42,25 @@ analysis.
...
-   should be a directory where these files will be placed. The optional argument
-   *context* is the number of lines of context to display around the current line
-   of source code in the traceback; this defaults to ``5``. If the optional
-   argument *format* is ``"html"``, the output is formatted as HTML.  Any other
-   value forces plain text output.  The default value is ``"html"``.
+   default handling for exceptions by setting the value of
+   :attr:`sys.excepthook`.
+
+   The optional argument *display* defaults to ``1`` and can be set to ``0``
+   to suppress sending the traceback to the browser. A message single line
```
Comment (mangrisano) [inline Doc/library/cgitb.rst]: Add another blank space after the full stop.

File: Doc/library/cgitb.rst [hunk 4]
```diff
@@ -42,16 +42,25 @@ analysis.
...
+   message declaring that "A problem has occurred ..." will continue to be
+   generated.
+
+   If the argument *logdir* is present, the traceback reports are written to
+   files.  The value of *logdir* should be a directory where these files will
+   be placed.
+
+   The optional argument *context* is the number of lines of context to
+   display around the current line of source code in the traceback; this
+   defaults to ``5``.
```
Comment (mangrisano) [inline Doc/library/cgitb.rst:59]: Here I would rather: 'the default value is ``5``. Same thing for the other ones in the file. IMHO, of course.

Comment (merwok) [reply Doc/library/cgitb.rst:59]: Why?  what’s the difference?

File: Doc/library/cgitb.rst (hunk 3 above)
Comment (merwok) [reply Doc/library/cgitb.rst]: Existing text does not seem to use that convention in this file!

File: Doc/library/cgitb.rst (hunk 4 above)
Comment (mangrisano) [reply Doc/library/cgitb.rst:59]: Perhaps there is no semantic difference between the two versions but I don't prefer yours. :)

File: Doc/library/cgitb.rst [hunk 5]
```diff
@@ -42,16 +42,25 @@ analysis.
    .. index:: single: excepthook() (in module sys)
 
    This function causes the :mod:`cgitb` module to take over the interpreter's
-   default handling for exceptions by setting the value of :attr:`sys.excepthook`.
-
-   The optional argument *display* defaults to ``1`` and can be set to ``0`` to
-   suppress sending the traceback to the browser. If the argument *logdir* is
-   present, the traceback reports are written to files.  The value of *logdir*
```
Comment (mangrisano) [inline Doc/library/cgitb.rst:49]: Here you can notice the convention I talked about previously.

File: Doc/library/cgitb.rst (hunk 2 above)
Comment (mangrisano) [reply Doc/library/cgitb.rst]: Here, before 'Any', another example of 'two blank space after full stop'.

File: Doc/library/cgitb.rst (hunk 4 above)
Comment (merwok) [reply Doc/library/cgitb.rst:59]: sorry what is my version?

Comment (mangrisano) [reply Doc/library/cgitb.rst:59]: Your version says "defaults to ``5``."

Comment (merwok) [reply Doc/library/cgitb.rst:59]: I am not the PR author!

Comment (mangrisano) [reply Doc/library/cgitb.rst:59]: Sorry, my mistake! :D I've just noticed that. Sorry, again.

File: Lib/test/test_sundry.py (hunk 1 above)
Comment (remilapeyre) [reply Lib/test/test_sundry.py]: Sorry about that

File: Doc/library/cgitb.rst (hunk 2 above)
Comment (remilapeyre) [reply Doc/library/cgitb.rst]: Thanks, I always forget about that.

File: Doc/library/cgitb.rst (hunk 4 above)
Comment (remilapeyre) [reply Doc/library/cgitb.rst:59]: Thanks for the feedback, I would like to keep as it is thought as `this defaults to ...` is already used elsewhere in this part of the documentation and I think consistency might be better.

**Comment by iritkatriel** [comment]
cgi/cgitb are deprecated as per PEP 594, so there won't be further enhancements to them.
